lib/Doctrine/ORM/Persisters/Entity/BasicEntityPersister.php

```diff
@@ -183,7 +183,7 @@ class BasicEntityPersister implements EntityPersister
      *
      * @var IdentifierFlattener
      */
-    private $identifierFlattener;
+    protected $identifierFlattener;
```

Comment:
```suggestion
    final protected $identifierFlattener;
```

Reply:
Why was this marked as resolved?

Reply:
Because of:

> PHP Fatal error:  Cannot declare property Doctrine\ORM\Persisters\Entity\BasicEntityPersister::$identifierFlattener final, the final modifier is allowed only for methods, classes, and class constants in /home/runner/work/orm/orm/lib/Doctrine/ORM/Persisters/Entity/BasicEntityPersister.php on line 186

Reply:
Oh right… don't know where I got that from!